services/gitlab/gitlab-issues.service.js

```diff
@@ -0,0 +1,446 @@
+import Joi from 'joi'
+import { optionalUrl } from '../validators.js'
+import { metric } from '../text-formatters.js'
+import GitLabBase from './gitlab-base.js'
+
+const schema = Joi.object({
+  statistics: Joi.object({
+    counts: Joi.object({
+      all: Joi.number().required(),
+      closed: Joi.number().required(),
+      opened: Joi.number().required(),
```

Comment:
These could all take advantage of our `nonNegativeInteger` helper that can be imported from the `validators.js` module located in the root `services` directory

```suggestion
      all: nonNegativeInteger,
      closed: nonNegativeInteger,
      opened: nonNegativeInteger,
```

Also I assume these fields will always exist even in a repository that's never had any issues?

Reply:
> Also I assume these fields will always exist even in a repository that's never had any issues?

yep